LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
What license did we use for pass?

Reply:
Updated license statement to whatever @htpvu said